### PR TITLE
Update 7.c

### PR DESCRIPTION
**Pull request by adidubs**
#### Checklist

- [ ] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by dubeyadi — APPROVED** [review]

**Review by dubeyadi — APPROVED** [review]

**Review by dubs25012002 — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
Could you explain the changes made in this file?

File: leetcode/src/7.c [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-    {
-        int pop = x % 10;
-        x /= 10;
-        if (rev > INT_MAX / 10 || (rev == INT_MAX / 10 && pop > 7))
-            return 0;
-        if (rev < INT_MIN / 10 || (rev == INT_MIN / 10 && pop < -8))
-            return 0;
-        rev = rev * 10 + pop;
+int reverse(int x){
+ int res=0;
```
Comment (xeland314) [inline leetcode/src/7.c:2]: Bad indentation.

File: leetcode/src/7.c [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+    
+    int s=1;
+    if(x > INT_MAX || x<(-INT_MAX)){
```
Comment (xeland314) [inline leetcode/src/7.c:5]: INT_MAX is undefined. Please add the header <limits.h>

**Review by xeland314 — CHANGES_REQUESTED** [review]
Please fix the highlighted sections

**Pull request closed without merging**